server/routes/convo.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const router = express.Router();
+const protect = require('../middleware/auth')
+
+const { createMessage, getAllConvos, getOneConvo } = require("../controllers/convo")
+
+router.route("/")
+.get(getAllConvos)
+.post(createMessage)
+
+router.route("/:friendId")
```

Comment:
If you won't be using the path param for this route, please remove it :)

Reply:
We will be using it. Changed controller to reflect route